### PR TITLE
make sure the ensure absent works with package install

### PR DESCRIPTION
**Pull request by fe80**
#### Pull Request (PR) description
Add ensure absent works for package installation.
I don't had this feature for the archive installation (maybe on a separated commit ?)

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

File: manifests/output.pp [hunk 1]
```diff
@@ -6,14 +6,27 @@
...
+#
+# [*ensure*]
+#   Set if the ensure params of the config file. If telegraf::ensure is absent the value is automatically absent
+#
 define telegraf::output (
-  String          $plugin_type = $name,
-  Optional[Array] $options     = undef,
+  String          $plugin_type                = $name,
+  Optional[Array] $options                    = undef,
+  Enum['present', 'absent'] $ensure = 'present',
```
Comment (bastelfreak) [inline manifests/output.pp:19]: can you please align the `=` here as well?

**Comment by bastelfreak** [comment]
@fe80 thanks for the PR. Can you please add a unit test for the change?

**Comment by fe80** [comment]
done

I had output spec (new file) on a separate commit :)

**Comment by bastelfreak** [comment]
thanks for the work!

**Pull request merged**